ZVV983/WeatherAnalyzer.Tests/Usings.cs

```diff
@@ -0,0 +1 @@
+global using NUnit.Framework;
```

Comment:
Ide pl beírhattad volna sok using-ot, ami a fájlokban jelen volt, csökkentve a vizuális zajt a tesztek elején.